lib/drivers/tandemDriver.js

```diff
@@ -1628,7 +1634,29 @@ module.exports = function (config) {
   };
 
   var buildCartridgeChangeRecords = function (data, records) {
-    var cartridgeRecords = filterLogEntries([PUMP_LOG_RECORDS.LID_CARTRIDGE_FILLED], data.log_records);
+    var cartridgeRecords = filterLogEntries([PUMP_LOG_RECORDS.LID_TUBING_FILLED,PUMP_LOG_RECORDS.LID_CANNULA_FILLED,PUMP_LOG_RECORDS.LID_CARTRIDGE_FILLED], data.log_records);
```

Comment:
Calling these all `cartridgeRecords` seems a bit odd to me since 2/3 are tubing records, but I wouldn't hold up a merge for it.